### DISCUSSION TITLE
Scrollable table body or sticky footer

### DISCUSSION THREAD

**Discussion by eafarooqi** [Power-Components/livewire-powergrid#557] [Q&A]
Hi, 

if we increase the per page number, The page height increases, how can we fix the table height and content inside the table should be scrollable. just like scrollY in js datatables. https://datatables.net/reference/option/scrollY

**Comment by akulmehta** [comment]
You can publish the views using

```
php artisan vendor:publish --tag=livewire-powergrid-views
```

Then edit the `table-base.blade.php` file and add custom classes to customize it.

**Reply by eafarooqi** [reply]
Okay Thanks, may be it can be added as an option or setting, so user can set the height of Y scroll.

**Reply by akulmehta** [reply]
Sure - can you make a PR for this, since you will be testing it out in your environment anyways?
